test/src/crypto.js

```diff
@@ -0,0 +1,149 @@
+import expect from 'expect.js';
+import sinon from 'sinon';
+import makeCryptoWith from '../../src/crypto';
```

Comment:
Tests should probably read from `lib`, since that's where the "compiled" version will live.